backend/azure/client.go

```diff
@@ -41,6 +41,9 @@ type Client interface {
 
```

Comment:
For this file, we don't have any unit tests so used the integration tests to run through each setting and have shared the verification details internally.